app/views/posts/show.html.erb

```diff
@@ -1 +1,18 @@
-<h1>Individual Posts Details</h1>
\ No newline at end of file
+<div class="post-container">
+   <div class="post-header">
+       <h2><%= @post.title%> By <%= @post.author.name%></h2>
+         <ul>
+             <li>Comments:<%= @post.comments_counter%></li>
+             <li>Likes:<%=@post.likes_counter%></li>
+         </ul>
+   </div>
+    <h4 class="comment-title"> Comments</h4>
+    <div class="post-body">
+       <%=@post.text%>
+    </div>
+     <div class="comment-container">
+        <%@post.comments.each do |comment|%>
+          <span> <%= comment.author.name%> :<%= comment.text%></span>
+          <%end%>
+     </div>
+</div>
```

Comment:
- [ ] [OPTIONAL] Since you've done everything to perfection, I can only mark this as optional. However, I noticed that on the posts show page, the post body is currently displayed under the header "Comments". In my opinion, this placement might confuse users, as they might perceive the post body as a comment, especially for shorter posts as shown below. To improve clarity, I suggest moving the "Comments" header down to line 13, allowing the post body to be clearly distinguished as the main content of the post.
- Here's how it looks:
  ![image](https://github.com/fatima-najafi/blog-app/assets/104968236/ac1b0ede-78f0-4921-9aba-0ba30a5e9454)